sdk/compute/Microsoft.Azure.Management.Compute/tests/SessionRecords/VMScenarioTests/TestVMScenarioOperations_DiffDisks.json

```diff
@@ -1,22 +1,22 @@
 {
```

Comment:
please take this out from the PR and keep the original.

Reply:
reverted to file in the 2021-04-01 branch